evi-modal-clm/modal_app.py

```diff
@@ -0,0 +1,15 @@
+from modal import Image, App, asgi_app
+from app import eliza_app
+
+# ------- MODAL --------
+# deploy with `poetry run python -m modal deploy modal_app.py`
```

Comment:
would be nice to also have a README for deploying to modal to parallel the nice cdk readme you added